sdk/eventhub/Azure.Messaging.EventHubs/src/Resources.resx

```diff
@@ -228,4 +228,7 @@
   <data name="RunningEventProcessorCannotPerformOperation" xml:space="preserve">
     <value>The event processor is already running and needs to be stopped in order to perform this operation.</value>
   </data>
+  <data name="ConsumerClientNeededForThisInformation" xml:space="preserve">
+    <value>The Event Hub Consumer client needed for this information</value>
```

Comment:
Maybe
`client needed` => `client is needed`
or
`The Event Hub...` => `Event Hub...`

Reply:
And we should add a period at the end.

Reply:
It's a fragment, intended as the subject of a sentence, so no period.   I don't know why I called out a consumer client, though.   Made it a bit more generic, as it should have been.  

Context:

`The Event Hub client responsible for this information has been closed and cannot perform the requested operation.`